packages/vats/src/core/chain-behaviors.js

```diff
@@ -419,7 +419,29 @@ export const produceHighPrioritySendersManager = async ({
   managerP.resolve(manager);
 };
 
+/** @param {BootstrapPowers & NamedVatPowers} powers */
+export const reflectAgoricNamesToChainStorage = async ({
+  consume: { chainStorage: rootP },
+  namedVat: {
+    consume: { agoricNames, board: vatBoard },
+  },
+}) => {
+  const root = await rootP;
+  if (!root) {
+    console.warn('cannot publish agoricNames without chainStorage');
```

Comment:
the sentence as written is true regardless of state.
```suggestion
    console.warn('no chainStorage: not publishing agoricNames');
```